### ISSUE TITLE
descDithers

### ISSUE THREAD

**Issue by humnaawan** [LSSTDESC/ObsStrat#3]
The idea is to allow access to descDithers (translational and rotational dithers; replace 
`fieldRA`, `fieldDec`, `rotTelPos`), similar to those implemented in DC1/DC2. For MAF users, this would translate to instructions to the right Stackers, while for non-MAF users, csv files with descDithers will be produced.

**Comment by rbiswas4** [comment]
@humnaawan Having the code here is great! Are the csv files obtained as a result of this available (say on nersc)?

**Comment by humnaawan** [comment]
@rbiswas4 yes, see [here](https://github.com/LSSTDESC/ObsStrat/blob/c8606dff00c0972630f9c8923f3d545f71918f37/code/descDithers/readme.md) for details, including the path to the directory with the csv files.

**Comment by rbiswas4** [comment]
@humnaawan Thanks. But I don't seem to have read permissions for either the databases or the csv files. Perhaps these could be put in a common area. I believe @heather999 has a DESC area we can use for such purposes, but I don't recall the location off the top of my head ... can check later today.

**Comment by humnaawan** [comment]
Just for documentation purposes, I'll note that the initial work was done in the `descDithers` branch and led to PR #2 and #4. I then deleted the branch, replacing it with [`issue/3/desc-dithers`](https://github.com/LSSTDESC/ObsStrat/tree/issue/3/desc-dithers), which follows the branching-naming convention we've decided upon to keep the branchs tied with specific Issues.

**Comment by humnaawan** [comment]
Based on the conversations at the Project & Community Workshop 2018, `baseline2018a` is the project-official baseline, not `kraken2026`. 

Keeping that in mind, I've put `baseline2018a.db` in `/global/cscratch1/sd/awan/dbs_wp_unzipped` and updated the readme (fee60f3beba7399a42d2581ae22df5e1606014c9). Then, I produced the csv file for the database (see below and 87e4d5aae690ce6fafde649316ad6de00bcf84e2  for details). @rbiswas4 please let me know if there are any problems.

---
To produce the dithers for only the one database, I changed the `descDiths_wp_cadences.py` file by using:
```
save_csv_dithers(dbs_path, outDir, db_files_only='baseline2018a.db', rot_rand_seed=42, trans_rand_seed=42,  print_progress=False, show_diagnostic_plots=False, save_plots=True)
```
but am not committing the changes since its a one-time thing only.

**Comment by rbiswas4** [comment]
@humnaawan 🥇

**Comment by humnaawan** [comment]
Update: the latest commits have implemented the less frequent translational dithering and file compression. The zipped files are now available in the same directory as before: `/global/homes/a/awan/desc/wp_descDithers_csvs`

@rbiswas4 @nregnault please let me know if there are any problems.

**Comment by rbiswas4** [comment]
@humnaawan 🥇  Trying them out tonight!
